akka-stream/src/main/scala/akka/stream/javadsl/OperationAttributes.scala

```diff
@@ -50,7 +50,9 @@ object OperationAttributes {
     private[akka] def asScala = scaladsl.OperationAttributes.dispatcher(dispatcher)
   }
 
-  private[akka] val none: OperationAttributes = new OperationAttributes {
+  def none() = _none
+
+  val _none: OperationAttributes = new OperationAttributes {
```

Comment:
Oh, and about making it public - I think it's useful for end users to have access to none sometimes, in code like `if (x) none else name(...)` etc.